src/utils/schema/widgetPropertyUpdaters.ts

```diff
@@ -256,12 +256,32 @@ function updateMinimum(
   updateProperty('minimum', value, verna, widgetPath, locale);
 }
 
+function updateDescription(
+  value: Properties['description'],
+  verna: VernaContextProps,
+  widgetPath: string[],
+  locale: string,
+) {
+  updateProperty('description', value, verna, widgetPath, locale);
+}
+
+function updateTitle(
+  value: Properties['title'],
+  verna: VernaContextProps,
+  widgetPath: string[],
+  locale: string,
+) {
+  updateProperty('title', value, verna, widgetPath, locale);
+}
```

Comment:
It's not DRY code, I think we have to improve this because we start to have numerous methods.